src/main/java/com/paneedah/weaponlib/crafting/CraftingEntry.java

```diff
@@ -67,4 +96,11 @@ public String toString() {
 		}
 	}
 
+	public double getYield(){
+		if(this.item instanceof ManufacturingItemBase && this.yield == 1){
+			return ((ManufacturingItemBase) this.item).getRecoveryChance();
+		}
+		return this.yield;
+	}
```

Comment:
What happened to the formatting, also remove the curly brackets on if statements that are one line

Reply:
removed curly braket on [1f43aa2](https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/303/commits/1f43aa26210a81a0627894a9a0dbaa87e0a1da56). I hope I managed to fix formatting too.